app/src/main/java/com/example/habitsmasher/LoginFragment.java

```diff
@@ -232,6 +380,20 @@ private void showMessage(String message, int length) {
                        length).show();
     }
 
+    @Override
+    public void onStart() {
+        super.onStart();
+        // Check if user is signed in (non-null) and update UI accordingly.
+
+        // note:  not working currently, but this is what we need to do if we want to persist
+        // the login session
+
+//        FirebaseUser currentUser = _auth.getCurrentUser();
+//        if (currentUser != null) {
+//            navigateToFragmentWithAction(R.id.action_navigation_login_to_HomeFragment);
+//        }
+    }
```

Comment:
Leaving this here, hopefully, address it during the last week of the project. If we decide that we want to "persist the login session" this is how we do it, and we may need to handle some flow issues from there on.